stryker4jvm-core/merge-notes-java-conversion.txt

```diff
@@ -0,0 +1,35 @@
+General note:
+there are no more dependencies on any scala library with the exception being 'fansi.Str'
+
+Thresholds:
+ScoreStatus has been made an enum and contains the determineScoreStatus method from scala's ThresholdChecker
+Scala's Thresholds has been moved from config package to run.threshold and converted to java
+
+Logger:
+Level.scala is converted to a java enum LogLevel;
+Scala's Logger trait has been converted to a java abstract Logger class;
+fansi.Str is still used by the java Logger
+SttpLogWrapper has not been converted as it should be implemented in stryker4jvm
+
+IgnoredMutationReason:
+Scala trait IgnoredMutationReason has been converted to java interface.
+Only the MutationExluded class has been added to core, the RegexParseError reason should be in scala-mutator
+
+Exception:
+Stryker4jvmException has been converted a java exception
+
+files:
+FileIO has been converted to a java interface;
+The class DisFileIO is reimplemented with pure java, in case that we still want IO[Unit] for fileIO in stryker4jvm
+we could implement a wrapper (should be possible).
+FileResolver has been ignored as it relies on fs2, this may instead be added to stryker4jvm
+
+Reporting:
+Converted scala Reporter to abstract java class without any IO (wrappers may be necessary in stryker4jvm if we wish to work with IO there);
```

Comment:
Could the reporters not also be inside stryker4jvm? They shouldn't be used by individual language mutators

Reply:
The merge-notes are outdated and are merely used during development of stryker4jvm to clarify where some things have been moved. They should not have ended up in the PR and will be removed.